oaknational/Oak-Web-Application
# [CUR-785] Phase-picker style changes

## Description

Music year: 2020

- Updated button styling
- Updated hover styling
- Updated drop down styling

## How to test

1. Go to https://deploy-preview-2748--oak-web-application.netlify.thenational.academy
2. Click on lot picker and select a subject
3. You should see the phase section matching the figma screenshots and designs [here](https://www.figma.com/design/3cz9fb0ObFDLv1c212OtJv/Final-designs---Curriculum-squad?node-id=4928-91083&node-type=INSTANCE&m=dev)
4. The tab focus behaviour should also display the updated styling

<img width="237" alt="Screenshot 2024-07-30 at 09 15 59" src="https://github.com/user-attachments/assets/325c94eb-e424-4a08-a00c-c3f27f1c1686">

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2024-09-02 at 11 25 49](https://github.com/user-attachments/assets/56aec8f9-6f44-4d4a-b4c2-ff80b9af6ccb)

How it should now look:
![Screenshot 2024-09-02 at 11 26 20](https://github.com/user-attachments/assets/6709ca78-264c-48a4-bc61-a09af3575220)

## Checklist
